zconf.h.in

```diff
@@ -50,24 +45,22 @@
  for small objects.
 */
 
-                        /* Type declarations */
+/* Type declarations */
 
 
 #ifndef OF /* function prototypes */
 #  define OF(args)  args
 #endif
 
-#if defined(WINDOWS) || defined(WIN32)
+#if defined(_WIN32)
```

Comment:
Borland C++ again...

Reply:
in what sense Borland? That `WINDOWS` define is nonsense. Most likely something that people saw that VS prop sheets add to gui apps and decided that it's the way to check for windows? 

No compiler defines `WINDOWS` or `_WINDOWS`: https://sourceforge.net/p/predef/wiki/OperatingSystems/


afaik gui apps on windows get _WINDOWS/WINDOWS added by VS prop sheets, but it's definitely not something that zlib needs to check. It's how to check for windows build

Reply:
Borland C does define ```WINDOWS```..., it is also mentioned few lines below...